recipe/meta.yaml

```diff
@@ -15,12 +15,13 @@ source:
     folder: tessdata
 
 build:
-  number: 0
+  number: 1000
```

Comment:
Maybe we should go to `1001` since we added the C++ compiler as well. That way we rebuild packages for the legacy compilers with this fix as well.

Reply:
With the new compilers, the build number is `1000`. However with the old compilers, we subtract `1000` so it is still `0`. The reason being we are trying to ensure the build numbers don't conflict between new and old compiler packages. Also this is done so that new compiler packages will take precedence over old compiler packages when the two sets of packages are merged back into the same `main` label.

Hope that helps. Please ask more questions if not.